google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/DmlBatchTest.java

```diff
@@ -63,23 +62,35 @@ private DmlBatch createSubject(UnitOfWork transaction) {
   @Test
   public void testExecuteQuery() {
     DmlBatch batch = createSubject();
-    exception.expect(SpannerExceptionMatcher.matchCode(ErrorCode.FAILED_PRECONDITION));
-    batch.executeQuery(mock(ParsedStatement.class), AnalyzeMode.NONE);
+    try {
+      batch.executeQuery(mock(ParsedStatement.class), AnalyzeMode.NONE);
+      fail("");
```

Comment:
nit: Here and below, please add a text like "Expected exception"

Reply:
Done